src/widgets-service.js

```diff
@@ -206,10 +222,35 @@ WidgetsService.prototype.createTimeSeriesModel = function (attrs, layer) {
   return widgetModel;
 };
 
+WidgetsService.prototype._dataviewModelAttrsNames = function (customAttrsNames) {
+  return [
+    'sync_on_data_change',
+    'sync_on_bbox_change',
+    'enabled'
+  ].concat(customAttrsNames);
+};
+
+WidgetsService.prototype._dataviewModelAttrs = function (allAttrs, customAttrs) {
+  return _.extend(
+    {
+      sync_on_data_change: _.isBoolean(allAttrs.sync_on_data_change)
```

Comment:
Won't this be handled by the dataview itself? (it has default values for these attributes).